content/api/subaccounts.apib

```diff
@@ -56,6 +56,9 @@ When making POST, PUT, or DELETE requests without the `X-MSYS-SUBACCOUNT` header
     + ip_pool (string)
         The ID of an IP Pool in which to restrict this subaccount's mail deliveries. If the supplied `ip_pool` is a empty or not present, this subaccount may use any of the account's IP Pools.
         <br/><a href="https://www.sparkpost.com/enterprise-email/"><span class="label label-warning"><strong>Enterprise</strong></span></a></strong> customers: please consult with your TAM on IP management.
+        + options (object)
```

Comment:
I think this one might have been indented too much. It should be the parent of deliverability, right?